site/components/live-editor/emotion-live-editor.tsx

```diff
@@ -24,14 +24,14 @@ const scope = {
         return require('@emotion/cache')
       case '@emotion/react':
         return require('@emotion/react')
-      case '@emotion/react/jsx-runtime':
-        return require('@emotion/react/jsx-runtime')
```

Comment:
This might still be needed if somebody uses the `/** @jsxImportSource @emotion/react */` pragma. Or is it not possible now?

Reply:
I was confused about what `@emotion/babel-plugin` does so the css prop was completely nonfunctional in the live editors... has been fixed now, including adding these 2 lines back to the fake `require` function.